app/assets/javascripts/components/components/status_content.jsx

```diff
@@ -39,7 +40,7 @@ const StatusContent = React.createClass({
       } else if (link.textContent[0] === '#' || (link.previousSibling && link.previousSibling.textContent && link.previousSibling.textContent[link.previousSibling.textContent.length - 1] === '#')) {
         link.addEventListener('click', this.onHashtagClick.bind(this, link.text), false);
       } else if (media) {
-        link.innerHTML = '<i class="fa fa-fw fa-photo"></i>';
+        link.innerHTML = '<Icon icon="photo" />';
```

Comment:
@Gargron It works. `<Icon>` references the icon component just like any of our other components
https://github.com/tootsuite/mastodon/pull/1562/files#diff-38b8991419d1039766d703c68c4a4ebdR3